src/compiler/emitter.ts

```diff
@@ -4563,7 +4582,14 @@ var __awaiter = (this && this.__awaiter) || function (thisArg, _arguments, Promi
                         emitLeadingComments(member);
                         emitStart(member);
                         emitStart((<MethodDeclaration>member).name);
-                        emitClassMemberPrefix(node, member);
+
+                        if (isStruct !== undefined) {

```

Comment:
This seems incorrect - even if you pass `false` for the optional boolean parameter you still do the alternate behavior.